packages/babel-helpers/scripts/generate-helpers.js

```diff
@@ -1,7 +1,7 @@
 import fs from "fs";
 import { join } from "path";
 import { URL, fileURLToPath } from "url";
-import { minify } from "terser"; // eslint-disable-line
+import { minify } from "terser"; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
We can add `{"devDependencies": ["**/*.test.js", "**/*.spec.js", "**/scripts/*.js"]}` to the eslint rule option.